mmv1/third_party/terraform/transport/config.go.tmpl

```diff
@@ -1481,6 +1488,35 @@ func (c *Config) GetCredentials(clientScopes []string, initialCredentialsOnly bo
 	return *creds, nil
 }
 
+// parse application default credentials to determine if they are X.509 certs
+func AreCredentialsX509() bool {
```

Comment:
Are ADCs the only expected way to use X509 credentials? Could a user not just point the "GOOGLE_CREDENTIALS" to the same file?

I'm thinking that we could support that, but not right now. So, maybe we should specify that this checks for ADC specifically. So like "CheckX509inADC"